Utils.ps1

```diff
@@ -32,9 +38,14 @@ function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     finally {
         [Console]::OutputEncoding = $currentEncoding
 
-        # Clear out stderr output that was added to the $Error collection
+        # Clear out stderr output that was added to the $Error collection, putting those errors in a module variable
         if ($global:Error.Count -gt $errorCount) {
-            $global:Error.RemoveRange(0, $global:Error.Count - $errorCount)
+            $numNewErrors = $global:Error.Count - $errorCount
+            $invokeErrors.InsertRange(0, $global:Error.GetRange(0, $numNewErrors))
+            if ($invokeErrors.Count -gt 256) {
+                $invokeErrors.RemoveRange(0, ($invokeErrors.Count - 256))

```

Comment:
A bit confused by this... wouldn't it make more sense to remove old errors from the bottom of the stack?

```powershell
$invokeErrors.RemoveRange(256, ($invokeErrors.Count - 256))
```